app/e2e/index.spec.ts

```diff
@@ -60,4 +61,45 @@ test.describe("Index page", () => {
     const displayPacketGroupName = getReadableIdString(packetGroupName);
     await expect(await getBreadcrumbLocator(page)).toHaveText(`home${displayPacketGroupName}${displayPacketId}`);
   });
+
+  test("can add update read permission on packet group", async ({ page }) => {
```

Comment:
You'll need to mark this test as not runnable on deployed instances since it changes the state of the system. You can do this using the `@stateMutate` [tag](@stateMutate) - there's a bit about this in the e2e section of the readme.